app/Http/Controllers/AppealController.php

```diff
@@ -93,6 +77,29 @@ public function appeal($id)
                 'previousAppeals' => $previousAppeals,
             ]);
         }
+
+        // if UTRS 1 table exists, let's check there as well
+        if (Schema::hasTable('oldappeals')) {
+            $info = Oldappeal::findOrFail($id);
+            $this->authorize('view', $info);
+
+            $comments = $info->comments;
+            $userlist = [];
+
+            foreach ($comments as $comment) {
+                if (!is_null($comment->commentUser) && !in_array($comment->commentUser, $userlist)) {
+                    $userlist[$comment->commentUser] = Olduser::findOrFail($comment->commentUser)->username;
+                }
```

Comment:
This is not a sufficient catch. With Oldappeals, Oldcomments has a zero equivalency being the system. Uploading a patch.